src/dataflow/components/dataflow-program.sass

```diff
@@ -10,12 +12,18 @@
     &.sensor, &.number, &.generator
       background-color: #0592AF
       border-color: #C0E4EB
+      &.selected
+        border-color: $color2-4
     &.logic, &.math, &.transform
       background-color: #2DA343
       border-color: #CDE8D2
+      &.selected
+        border-color: $color2-4
     &.relay, &.data-storage
       background-color: #E16D2F
       border-color: #E09E7D
+      &.selected
+        border-color: $color2-4
```

Comment:
If the selected color is the same for all nodes, couldn't it be set once for `.node` rather than separately for each set of types?

Reply:
I originally had these setting unique colors for each block on selection, but it looked weird.  Better to wait and have Michael think about that sort of detail.

Reply:
Fixed.